acme/tests/messages_test.py

```diff
@@ -430,12 +418,15 @@ class CertificateRequestTest(unittest.TestCase):
     def setUp(self):
         from acme.messages import CertificateRequest
         self.req = CertificateRequest(csr=CSR)
+        # 'resource' gets stripped as a non-compliant field on serialization: restore it.
+        self.req_json = self.req.to_json()
+        self.req_json['resource'] = 'new-cert'
```

Comment:
Looking back at this, are we just putting it back in because we have some old test data around that we're comparing to or something? What still wants a resource field with old-style `new-cert`?

Reply:
Great, we can revert this.